### PR TITLE
Use and document correct `lazyLoading.enabled` invocation.

### PR DESCRIPTION
**Pull request by chriskrycho**
Per [the current implementation][src], the correct invocation is not a boolean but an object with `enabled: boolean` on it. This PR updates both the engine itself and the guides to match that.

[src]: https://github.com/ember-engines/ember-engines/blob/1265ad153f724c531da5e42df47c52fc9f9c71da/lib/engine-addon.js#L384:L390

### CODE REVIEW

File: lib/guide/index.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
 /*jshint node:true*/
-const EngineAddon = require('ember-engines/lib/engine-addon');
+const EngineAddon = require("ember-engines/lib/engine-addon");
 module.exports = EngineAddon.extend({
```
Comment (rwjblue) [inline lib/guide/index.js]: These formatting changes seem unrelated, can you revert them?

**Review by rwjblue — CHANGES_REQUESTED** [review]

File: lib/guide/index.js (hunk 1 above)
Comment (chriskrycho) [reply lib/guide/index.js]: Whoops! Yes, sorry – thought I'd caught and reverted all of those already!

Comment (chriskrycho) [reply lib/guide/index.js]: Done and force pushed.

**Comment by villander** [comment]
@rwjblue can you merge this, please?

File: lib/guide/index.js [hunk 2]
```diff
@@ -7,5 +7,8 @@ module.exports = EngineAddon.extend({
     return true;
   },
 
-  lazyLoading: true
+  // eslint-disable-next-line ember/avoid-leaking-state-in-ember-objects
```
Comment (rwjblue) [inline lib/guide/index.js]:
```suggestion
```

File: lib/guide/index.js [hunk 3]
```diff
@@ -7,5 +7,8 @@ module.exports = EngineAddon.extend({
...
+  lazyLoading: {
```
Comment (rwjblue) [inline lib/guide/index.js]:
```suggestion
  lazyLoading: Object.freeze({
```

File: lib/guide/index.js [hunk 4]
```diff
@@ -7,5 +7,8 @@ module.exports = EngineAddon.extend({
...
+    enabled: true
+  }
```
Comment (rwjblue) [inline lib/guide/index.js]:
```suggestion
  })
```

**Review by rwjblue — COMMENTED** [review]
Made some suggestions, but if we freeze we don't need to disable the linting rule inline.

**Comment by chriskrycho** [comment]
Updated, @rwjblue – thank you!

**Review by rwjblue — APPROVED** [review]

**Comment by rwjblue** [comment]
Awesome, thank you!
